### PR TITLE
Fix OnShowPoolCommand after Mempool changes

### PR DESCRIPTION
**Pull request by vncoelho**

### CODE REVIEW

**Comment by jsolman** [comment]
Neo 2.9.5 will need to get released first and get referenced in this project also so that this will have access to the new MemoryPool methods before this can be merged.

**Comment by erikzhang** [comment]
You can use [MyGet](https://www.myget.org/feed/neo/package/nuget/Neo) daily build version for testing.

**Comment by vncoelho** [comment]
I already tested locally, Erik, with https://github.com/NeoResearch/neo-tests/blob/master/docker-build-neo-cli/env-repositories-improved-dbft.sh.
It is working.

However, I got some strange Akka red messages, probably related to the mempool changes, I need to double check.

File: neo-cli/Shell/MainService.cs [hunk 1]
```diff
@@ -810,11 +810,19 @@ private bool OnShowCommand(string[] args)
...
-            Transaction[] transactions = Blockchain.Singleton.GetMemoryPool().ToArray();
             if (verbose)
-                foreach (Transaction tx in transactions)
+            {
+                Blockchain.Singleton.MemPool.GetVerifiedAndUnverifiedTransactions(
+                    out IEnumerable<Transaction> verifiedTransactions, 
+                    out IEnumerable<Transaction> unverifiedTransactions);                
+                Console.WriteLine("Verified Transactions:");
+                foreach (Transaction tx in verifiedTransactions)
+                    Console.WriteLine($"{tx.Hash} {tx.GetType().Name}");
```
[the remark this answers is not in the document]
Comment (vncoelho) [reply neo-cli/Shell/MainService.cs]: Can you adjust this, @jsolman, I did not get exactly...aehauhuea

**Comment by jsolman** [comment]
@vncoelho The error you got may have been related to my fix today neo-project/neo/pull/549

File: neo-cli/Shell/MainService.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (jsolman) [reply neo-cli/Shell/MainService.cs]: Done.

**Comment by vncoelho** [comment]
Perfect, Jeff, I tested here and errors disappeared.

This PR is also correct.
Maybe it is a good time to release 2.9.5, because commit phase will kind of require everyone to update their protocol and the current changes are softer.

**Comment by jsolman** [comment]
I agree with releasing 2.9.5 ASAP, since it should help ensure a reliable network.

**Comment by vncoelho** [comment]
There is a pre-release here if needed: https://github.com/NeoResearch/neo-tests/blob/master/releases/neo-release-2.9.5-all-plugins-improved-dbft.zip

**Pull request merged**